portfolio/src/main/webapp/index.html

```diff
@@ -60,6 +60,16 @@ <h3>Comments:</h3>
         <option value="25">25</option>
         <option value="50">50</option>
       </select>
+      <label for="language">Language:</label>
+      <select name="language" id="language" onchange="getComments()">
```

Comment:
Consider making the id more specific, because it has to be unique.

ex:
language-select

Reply:
Done.